### ISSUE TITLE
Missing "./configure" may end up with weird error.

### ISSUE THREAD

**Issue by ctheune** [zopefoundation/zc.recipe.cmmi#1]
I stumbled over this:

```
Traceback (most recent call last):
  File "/Users/ctheune/Code/baby-pi-mobile/lib/python2.7/site-packages/zc/buildout/buildout.py", line 1992, in main
    getattr(buildout, command)(args)
  File "/Users/ctheune/Code/baby-pi-mobile/lib/python2.7/site-packages/zc/buildout/buildout.py", line 666, in install
    installed_files = self[part]._call(recipe.install)
  File "/Users/ctheune/Code/baby-pi-mobile/lib/python2.7/site-packages/zc/buildout/buildout.py", line 1407, in _call
    return f()
  File "/Users/ctheune/Code/baby-pi-mobile/eggs/zc.recipe.cmmi-1.3.6-py2.7.egg/zc/recipe/cmmi/__init__.py", line 113, in install
    self.build()
  File "/Users/ctheune/Code/baby-pi-mobile/eggs/zc.recipe.cmmi-1.3.6-py2.7.egg/zc/recipe/cmmi/__init__.py", line 193, in build
    os.chdir(entries[0])
OSError: [Errno 2] No such file or directory: 'opencv-3.0.0'
```

It turns out that in 
https://github.com/zopefoundation/zc.recipe.cmmi/blob/master/src/zc/recipe/cmmi/__init__.py#L193 the os.chdir() has already happened and thus we're already in the directory it wants to jump to. This is shadowing the fact that configure doesn't exist in this archive.

(I was trying to build OpenCV 3.0)

**Comment by jamadden** [comment]
PR #2 incidentally addresses this.

**Comment by jamadden** [comment]
Fixed in #2.
